.github/workflows/check-catalyst.yaml

```diff
@@ -545,11 +545,18 @@ jobs:
         name: runtime-build-${{ matrix.compiler }}
         path: runtime-build/lib
 
+    - name: Download OQD-Runtime Artifact
+      uses: actions/download-artifact@v4
+      with:
+        name: oqd-build-${{ matrix.compiler }}
+        path: oqd-build
```

Comment:
TODO: This was necessary to get the OQD tests in the pytest suite to pass with the lightning-kokkos backend. Do we need to run the OQD tests with lightning-kokkos? Probably not; can we skip these tests depending on the backend?

Reply:
8fb6664f